datadog_checks_base/datadog_checks/base/utils/http.py

```diff
@@ -313,6 +313,9 @@ def patch(self, url, **options):
     def delete(self, url, **options):
         return self._request('delete', url, options)
 
+    def options_method(self, url, **options):
```

Comment:
Let's name that `options` to match how `requests` names it

Reply:
Thanks for taking a look Florian! I named it `options_method` so that it wouldn't conflict with the `self.options` that already exists and cause a `TypeError: 'dict' object is not callable`: https://github.com/DataDog/integrations-core/blob/2b3d4c968d7e85fc81434219a83251c03b06294c/datadog_checks_base/datadog_checks/base/utils/http.py#L259-L266

Are there ways I can follow convention and name it `options` without conflicting with `self.options` or should I keep it as is? I considered re-naming `self.options`, but I was concerned about re-naming it across the codebase and introducing bugs. Let me know what you think.

Reply:
Hadn't thought of that, that's a bit unfortunate and we'd have to rename `options` in integrations as well + custom checks may rely on this. So let's go with `options_method` then.